tests/test_delete_customer.py

```diff
@@ -0,0 +1,52 @@
+import allure
+import pytest
+
+from pages.customers_page import CustomersPage
+from pages.manager_page import ManagerPage
+from utils import filter_names_to_delete
+
+
+@allure.feature("XYZ Bank")
+@allure.story('UI')
+@allure.title('Удаление клиента')
+@allure.description("""
+    Цель: Проверить удаление клиента из таблицы
+
+    Предусловие:
+        - открыть браузер
+        - добавить нового клиента 
+
+    Шаги:
+    1. Перейти на страницу XYZ Bank
+    2. Нажать кнопку 'Customers'
+    3. Определить имена для удаления, согласно алгоритма:
+        - Получить из таблицы Customers список имен 
+        - Узнать длину каждого имени
+        - Найти среднее арифметическое получившихся длин 
+        - Удалить клиента с тем именем, у которого длина будет ближе к среднему арифметическому
+    4. Убедиться, что после удаления в таблице остались записи, которые не попали под условия алгоритма
+    
+    Постусловие:
+        - Закрыть браузер
+    
+    Ожидаемый результат:
+        -  из таблицы удалена запись
+    """)
+@pytest.mark.ui
+def test_delete_customer(driver, add_customer):
+
+    with allure.step('Открытие страницы XYZ bank, добавление нового клиента, переход на страницу с клиентами'):
+        page = ManagerPage(driver)
```

Comment:
Здесь не очень понятно с какой страницей мы работаем.
```suggestion
        manage_page = ManagerPage(driver)
```